### PR TITLE
Fix right way

### PR DESCRIPTION
**Pull request by gonzakpo**
right way:
http://php.net/manual/en/control-structures.switch.php

$actionName = 'automplete'
error: no break changes $actionName to 'list'

### CODE REVIEW

**Comment by alterphp** [comment]
This is the expected behavior, we want `autocomplete` action to be mapped on `list` action.

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -34,6 +34,7 @@ protected function isActionAllowed($actionName)
         switch ($actionName) {
             // autocomplete action is mapped to list action for access permissions
```
Comment (alterphp) [inline src/Controller/AdminController.php]: the comment is clear ;-)

**Comment by gonzakpo** [comment]
Example role security does not work:
easy_admin:
     entities:
         Entity:
             role: ROLE_SUPER_ADMIN
             list:
                 ...
             autocomplete:
                 role: ROLE_USER

File: src/Controller/AdminController.php (hunk 1 above)
Comment (gonzakpo) [reply src/Controller/AdminController.php]: see the list does not necessarily require the same permissions as see the autocomplete data

Comment (gonzakpo) [reply src/Controller/AdminController.php]: @alterphp They are different

**Comment by alterphp** [comment]
Of course, there is no role for autocomplete, it's mapped on role required for list.

Do you really need different role for autocomplete and list ?

**Comment by gonzakpo** [comment]
By default it takes the entity role, as in the example, but a minor role does not see a select autcomplete in a form

**Comment by alterphp** [comment]
Autocomplete required role for `list` as autocomplete basically list items. Give the role required for list and it will work.

**Comment by gonzakpo** [comment]
If
```
list:
    role: ROLE_USER
```
You can access:
http://127.0.0.1:8000/admin/?action=list&entity=Entity

If
```
autocomplete:
    role: ROLE_USER
```
Do not access:
http://127.0.0.1:8000/admin/?action=list&entity=Entity

**Comment by alterphp** [comment]
So you want the same user cannot access list but is allowed for autocomplete ?

This is not expected by the bundle and changing it is a BC break.

File: src/Controller/AdminController.php (hunk 1 above)
Comment (alterphp) [reply src/Controller/AdminController.php]: Why not, we all have different ways to structure role hierarchies. This is fortunately customizable to your needs by overriding `isActionAllowed` method.

**Comment by gonzakpo** [comment]
If I treat the role 'autocomplete' only if it is defined in the yaml, otherwise I will use it as 'list'.
can be?

**Comment by alterphp** [comment]
It would be the good move, but required changes in the code seems important for a minor improvement (that can easily be obtained by overriding).

**Pull request closed without merging**